utils/src/main/kotlin/Extensions.kt

```diff
@@ -370,6 +370,12 @@ fun String.stripCredentialsFromUrl() =
         this
     }
 
+/**
+ * If this string starts with [prefix], return the string without the prefix, otherwise return null.
+ */
+fun String?.withoutPrefix(prefix: String): String? =
```

Comment:
Consider adding a second argument like e.g. `missingPrefixValue: String? = null`. It may lead to a bit more readable caller code.

Reply:
Done.